### PR TITLE
MOTECH-1710: ScheduleTracking javadoc events package

### PR DESCRIPTION
**Pull request by tstalka**

### CODE REVIEW

**Comment by tstalka** [comment]
Test this please

File: schedule-tracking/src/main/java/org/motechproject/scheduletracking/events/EnrolledUserEvent.java [hunk 1]
```diff
@@ -10,26 +10,52 @@
...
+
     private LocalDate enrollmentDate;
+
     private Time enrollmentTime;
+
     private String startingMilestoneName;
 
-    public EnrolledUserEvent(String externalId, String scheduleName, Time preferredAlertTime, DateTime referendeDateTime, DateTime enrollmentDateTime, String startingMilestoneName) {
+    /**
+     * Creates a EnrolledUserEvent with the externalId attribute set to {@code scheduleName}, the scheduleName attribute to {@code scheduleName},
```
Comment (kmadej) [inline schedule-tracking/src/main/java/org/motechproject/scheduletracking/events/EnrolledUserEvent.java]: ...externalId attribute set to code externalId...

File: schedule-tracking/src/main/java/org/motechproject/scheduletracking/events/MilestoneDefaultedEvent.java [hunk 2]
```diff
@@ -7,23 +7,50 @@
 
 import java.util.HashMap;
 
+/**
+ * The <code>EnrolledUserEvent</code> is used to create a Motech event. The event payload contains details about milestone
```
Comment (kmadej) [inline schedule-tracking/src/main/java/org/motechproject/scheduletracking/events/MilestoneDefaultedEvent.java]: The MilestoneDefaultedEvent

File: schedule-tracking/src/main/java/org/motechproject/scheduletracking/events/UnenrolledUserEvent.java [hunk 3]
```diff
@@ -7,15 +7,33 @@
...
+ * @see org.motechproject.scheduletracking.service.EnrollmentService
+ */
 public class UnenrolledUserEvent {
+
     private String externalId;
+
     private String scheduleName;
 
+    /**
+     * Creates a UnenrolledUserEvent with the externalId attribute set to {@code scheduleName},
```
Comment (kmadej) [inline schedule-tracking/src/main/java/org/motechproject/scheduletracking/events/UnenrolledUserEvent.java]: set to externalId

File: schedule-tracking/src/main/java/org/motechproject/scheduletracking/events/UnenrolledUserEvent.java [hunk 4]
```diff
@@ -7,15 +7,33 @@
...
+     * @param externalId the user external id
+     * @param scheduleName the name of the schedule
+     */
     public UnenrolledUserEvent(String externalId, String scheduleName) {
         this.externalId = externalId;
         this.scheduleName = scheduleName;
     }
 
+    /**
+     * Creates a UnenrolledUserEvent from map with parameters.
```
Comment (kmadej) [inline schedule-tracking/src/main/java/org/motechproject/scheduletracking/events/UnenrolledUserEvent.java]: with the given parameters

**Pull request merged**